pkg/apis/edgefs.rook.io/v1/types.go

```diff
@@ -76,8 +76,10 @@ type ClusterSpec struct {
 	SystemReplicationCount  int               `json:"sysRepCount,omitempty"`
 	FailureDomain           string            `json:"failureDomain,omitempty"`
 	CommitNWait             int               `json:"commitNWait,omitempty"`
+	NoIP4Frag               bool              `json:"noIP4Frag,omitempty"`
 	MaxContainerCapacity    resource.Quantity `json:"maxContainerCapacity,omitempty"`
 	UseHostLocalTime        bool              `json:"useHostLocalTime,omitempty"`
+	SysChunkSize            int               `json:"sysChunkSize,omitempty"`
```

Comment:
Probably rename it to SysMaxChunkSize and document?

Reply:
Renamed and documented